sir_cake/sir_cake/urls.py

```diff
@@ -22,8 +22,9 @@
 
 urlpatterns = [
     path('new-product/', seller_views.new_product, name="new-product"),
+    path('item/<int:pk>/', seller_views.item_detail, name="item_detail"),
```

Comment:
Is `item` and `product` the same? If so, let's make the urls and view names consistent, e.g. `new_product` and `product_detail`.